engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-versions.controller.js

```diff
@@ -170,7 +170,9 @@ angular.module('Bastion.content-views').controller('ContentViewVersionsControlle
 
         $scope.status = function (version) {
             var taskTypes = $scope.taskTypes,
-                deletionEvents = findTaskTypes(version['active_history'], taskTypes.deletion),
+                deletionEvents = _.filter(findTaskTypes(version['active_history'], taskTypes.deletion), function(history) {
+                    return history.environment != null;
+                }),
                 promotionEvents = findTaskTypes(version['active_history'], taskTypes.promotion),
```

Comment:
Is the same problem present for promote, publish?

Reply:
No, the same problem is not present. I tested both, as @omkarkhatavkar did.